pkgs/applications/virtualization/docker/default.nix

```diff
@@ -181,14 +182,13 @@ rec {
     nativeBuildInputs = [
       makeWrapper pkg-config go-md2man go libtool installShellFiles
     ];
-    buildInputs = lib.optional (!clientOnly) sqlite
-      ++ lib.optional withLvm lvm2
-      ++ lib.optional withBtrfs btrfs-progs
-      ++ lib.optional withSystemd systemd
-      ++ lib.optional withSeccomp libseccomp
-      ++ plugins;
-
-    patches = [
+
+    buildInputs = plugins ++ lib.optionals (lib.versionAtLeast version "23") [
```

Comment:
This doesn't work on `aarch64-darwin`:

* https://github.com/NixOS/nixpkgs/pull/264461#issuecomment-1813291023